openquake/server/settings.py

```diff
@@ -22,8 +22,7 @@
 from openquake.baselib import config, datastore
 
 try:
-    import openquakeplatform
-    STANDALONE = True
+    from openquakeplatform.settings import STANDALONE, STANDALONE_APPS
```

Comment:
`STANDALONE_APPS` must exist anyway in this configuration, because it's used here https://github.com/gem/oq-engine/blob/d6d8c6f3d157e7375ae40ae567d600d16eaa73d8/openquake/server/urls.py#L39

So, or we do something like:
```python
STANDALONE = False
STANDALONE_APPS = ()

from openquakeplatform import settings as standalone_settings

STANDALONE = standalone_settings.STANDALONE
STANDALONE_APPS = standalone_settings.STANDALONE_APPS
```

or we have to change `urls.py` too

Reply:
Read list of web applications from openquakeplatform.settings.